src/list/FilterSidebar.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react';
+
+import styled from 'styled-components';
+import { Label, SearchInput, CheckboxGroup, IndeterminateCheckbox } from '@sailthru/stui-components';
+import { fontSizeSmall, fontSizeLarge, fontWeightBold, grayTint75 } from '@sailthru/design-tokens';
+
+const FilterContainer = styled.div`
+    margin-top: 1.875em;
+    background-color: ${grayTint75};
+    padding: 1em;
+    height: 100vh
+`;
+
+const OptionGroupContainer = styled.div`
+    margin-top: 1em;
+`;
+
+export default function FilterSidebar({ onChange }) {
+    const [ searchValue, setSearchValue ] = useState('');
+    const [ selectedMode, setSelectedMode ] = useState([]);
+    const [ selectedType, setSelectedType ] = useState([]);
+    const [ modesList, setModesList ] = useState([]);
+    const [ typesList, setTypesList ] = useState([]);
+
+    const modes = ['HTML', 'Visual'];
+    const types = ['Signup', 'Manage', 'Other'];
+    
+    function onChangeMode({newValue}) {
+        setSelectedMode(newValue);
+        const modeList = [];
+        newValue.forEach(value => {
+            modeList.push(modes[value]);
+        })
+        setModesList(modeList);
+        onChange({types: typesList, modes: modeList, name: searchValue});
+    }
+
+    function onChangeType({newValue}) {
+        setSelectedType(newValue);
+        const typeList = [];
+        newValue.forEach(value => {
+            typeList.push(types[value]);
+        })
+        setTypesList(typeList);
+        onChange({types: typeList, modes: modesList, name: searchValue});
+    }
+
+    function onChangeSearchValue(newValue) {
+        setSearchValue(newValue);
+        onChange({types: typesList, modes: modesList, name: newValue});
+    }
+
+    return (
+        <FilterContainer>
+            <OptionGroupContainer>
+                <Label fontsize={fontSizeLarge} style={{fontWeight: fontWeightBold}}>
+                    FILTER
+                </Label>
+                <SearchInput
+                    value={searchValue}
+                    placeholder="Search"
+                    onChange={onChangeSearchValue}
+                />
+            </OptionGroupContainer>
+            <OptionGroupContainer>
+                <Label fontsize={{ fontSizeSmall }} style={{ fontWeight: fontWeightBold }}>
+                    Mode
```

Comment:
Blocking: Even though internally this is mode, externally (with customers) its supposed to be type
```suggestion
                    Type
```

Reply:
Should be fixed.